### ISSUE TITLE
The project system 'ScriptProjectSystem' threw exception during initialization.

### ISSUE THREAD

**Issue by TzeWey** [OmniSharp/omnisharp-roslyn#682]
Using `dev@dabe14b`, I tried force loading my folder without a project.json file and expect the new ScriptProjectSystem to use the 'non-context usage' mode. 
However, an exception is thrown during initialization instead.

Log output below:
```
[ERROR:OmniSharp.Startup] The project system 'ScriptProjectSystem' threw exception during initialization.
Value cannot be null.
Parameter name: source
   at System.Linq.Enumerable.First[TSource](IEnumerable`1 source)
   at OmniSharp.Script.ScriptProjectSystem.Initalize(IConfiguration configuration) in \src\OmniSharp.Script\ScriptProjectSystem.cs:line 81
   at OmniSharp.Startup.Configure(IApplicationBuilder app, IServiceProvider serviceProvider, IOmnisharpEnvironment env, ILoggerFactory loggerFactory, ISharedTextWriter writer, IOmnisharpAssemblyLoader loader, IOptions`1 optionsAccessor) in \src\OmniSharp.Host\Startup.cs:line 183
```

The below check would never be true if a project.json file is not found because `runtimeContexts` would be `null`
```
if (runtimeContexts?.Any() == false) 
{
}
```

Swapping the 'if' statement around to check for `true` instead should resolve this, if will create a pull request if the fix is good. Thanks!

**Comment by DustinCampbell** [comment]
cc @filipw

**Comment by filipw** [comment]
the check should say `if (runtimeContexts == null || runtimeContexts.Any() == false)` - that will fix the error. It was  mistakenly changed [here](https://github.com/OmniSharp/omnisharp-roslyn/commit/11f9e1039978338fb5a47c93e7e6bcca7cf3e71c#diff-ed5a3c116a19423c18b1e630f80455b9L69)

The reason is: if there is no runtime context because there is no `project.json` (the `null` path) or if there is no runtime context because `project.json` doesn't contain the runtime definition (the `Any() == false` path), we should default to the non-context path.

@TzeWey if you can create the PR, that would be great, otherwise I'll do it later. Another note is that OmniSharp will not light up without `sln/csproj/project.json` at the moment 😄  so the only way to use it with scripting without `project.json` is to open a dotnet-cli project first, let OmniSharp start, and then switch folder.

**Comment by DustinCampbell** [comment]
Twas my bad in CR feedback. :smile:

**Comment by filipw** [comment]
Fixed by #684 - thanks @TzeWey
